opm/parser/eclipse/EclipseState/Schedule/Completion.hpp

```diff
@@ -89,10 +93,12 @@ namespace Opm {
         Value<double> m_connectionTransmissibilityFactor;
         double m_wellPi;
         Value<double> m_skinFactor;
+        Value<int> m_satTableId;
```

Comment:
Should not be necessary to use the `Value<int>` type now, since we are guaranteed that the value will be correctly initialized in the constructor.